arkouda/dataframe.py

```diff
@@ -973,6 +1150,29 @@ def drop(
         >>> df.drop(1)
         or
         >>> df.drop(1, axis=0)
```

Comment:
Right above this, we need to change it to be something like the below for to fix how it is displayed
```python
>>> df.drop('col_name', axis=1)  # Drop column
>>> df.drop(1)   # Drop Row
>>> df.drop(1, axis=0)
```

Reply:
Changed it.